unity-renderer/Assets/Scripts/MainScripts/DCL/Models/AvatarModel/AvatarModel.cs

```diff
@@ -101,15 +102,20 @@ public void CopyFrom(AvatarModel other)
         skinColor = other.skinColor;
         hairColor = other.hairColor;
         eyeColor = other.eyeColor;
-        expressionTriggerId = other.expressionTriggerId;
-        expressionTriggerTimestamp = other.expressionTriggerTimestamp;
+        if (!string.IsNullOrEmpty(other.expressionTriggerId) && expressionTriggerId != other.expressionTriggerId)
```

Comment:
i think there may be a responsibility problem on these new checks. I think that `CopyFrom` method should copy all the attributes, because this is usage dependent implementation. These checks should be handled elsewhere

Reply:
Done!